homeassistant/components/romy/sensor.py

```diff
@@ -0,0 +1,109 @@
+"""Sensor checking adc and status values from your ROMY."""
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    AREA_SQUARE_METERS,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    EntityCategory,
+    UnitOfLength,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import RomyVacuumCoordinator
+from .entity import RomyEntity
+
+SENSORS: list[SensorEntityDescription] = [
+    SensorEntityDescription(
+        key="battery_level",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.BATTERY,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key="rssi",
+        entity_registry_enabled_default=False,
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key="dustbin_sensor",
+        translation_key="dustbin_sensor",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key="total_cleaning_time",
+        translation_key="total_cleaning_time",
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key="total_number_of_cleaning_runs",
+        translation_key="total_number_of_cleaning_runs",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="Cleaning Runs",
```

Comment:
```suggestion
        native_unit_of_measurement="cleaning runs",
```

Reply:
Maybe consider just using "runs" since the entity name already contains "cleaning"

Reply:
changed to Runs